assets/sass/components/setup/_googlesitekit-setup-module.scss

```diff
@@ -126,9 +126,22 @@
 		> .googlesitekit-setup-module__input {
 			margin: 1em 0;
 		}
+	}
+
+	div[name="optimizeID"] {
+		margin-top: 20px;
+		width: 100%;
+
+		@media (min-width: $bp-wpAdminBarTablet) {
+			min-width: 350px;
+			width: auto;
+		}
+
+		+ .mdc-text-field-helper-line {
 
-		+ .googlesitekit-error-text {
-			margin-top: -24px;
+			.mdc-text-field-helper-text {
+				font-size: 0.75rem;
```

Comment:
Looks like this is the correct font-size for helper-text but we should be applying it consistently throughout rather than only here; let's address that in another issue though as it's beyond the scope of this one.